src/common/SkeletonLoader.tsx

```diff
@@ -0,0 +1,45 @@
+import { JSX, useMemo } from 'react';
+import { Placeholder } from 'react-bootstrap';
+
+interface SkeletionLoaderProps {
+  columnsCount: number;
+  rowsCount: number;
+}
+
+export default function SkeletonLoader({
+  columnsCount,
+  rowsCount,
+}: SkeletionLoaderProps) {
+  const columns = useMemo(() => {
+    const skeletons: JSX.Element[] = [];
+    for (let i = 0; i < columnsCount; i++) {
+      skeletons.push(
+        <Placeholder
+          bg="light"
+          as={'td'}
+          animation="glow"
+          key={`column-${i}`}
+          size="lg"
+          style={{ flex: 1 }}
+        >
+          <Placeholder
+            bg="secondary"
+            xs={12}
+            style={{ height: '36px', borderRadius: 6 }}
+          />
+        </Placeholder>
+      );
+    }
+    return skeletons;
+  }, [columnsCount]);
```

Comment:
```suggestion
  const columns = useMemo(() => {
    const skeletons = Array.from({ length: columnsCount }).map((_, i) => (
      <Placeholder
        bg="light"
        as={'td'}
        animation="glow"
        key={`column-${i}`}
        size="lg"
        style={{ flex: 1 }}
      >
        <Placeholder
          bg="secondary"
          xs={12}
          style={{ height: '36px', borderRadius: 6 }}
        />
      </Placeholder>
    ));
    return skeletons;
  }, [columnsCount]);
```
@rhehfl 반복을 위한 count만 주어졌을 때 배열 메서드를 활용이 필요 할 경우 `Array.from`을 활용해봐도 좋을 것 같습니다. JSX.Element[] 타입도 추론이 잘되고, push, for문을 제외할 수 있습니다.

Reply:
@ssi02014 먼저 좋은 리뷰 감사합니다! 평소에 많이 사용해본 메서드가 아니라서 문서를 찾아봤는데 [Array.from()](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/from#%ED%99%94%EC%82%B4%ED%91%9C_%ED%95%A8%EC%88%98%EC%99%80_array.from_%EC%82%AC%EC%9A%A9%ED%95%98%EA%B8%B0) 두번째 인자로 바로 콜백을 받을 수 있더라구용
```ts
 const columns = useMemo(() => {
    const skeletons = Array.from({ length: columnsCount }, (_, index) => (
      <Placeholder
        bg="light"
        as={'td'}
        animation="glow"
        key={`column-${index}`}
        size="lg"
        style={{ flex: 1 }}
      >
        <Placeholder
          bg="secondary"
          xs={12}
          style={{ height: '36px', borderRadius: 6 }}
        />
      </Placeholder>
    ));

    return skeletons;
  }, [columnsCount]);
```
그래서 이런 방식으로 작성해봤는데 어떤가요 ?!🤔🤔

Reply:
전혀 문제없습니다😁